### PR TITLE
Fixed alignment for CTA buttons in PWA.

### LINKED ISSUE #5025: [CAL-160] fix cta buttons on full-screen pwa mode
> From [Linear-GitHub Sync](https://synclinear.com)

### PR DESCRIPTION
**Pull request by JeroenReumkens**
## What does this PR do?

Fixes the alignment of the CTA buttons in shall wen used in PWA mode
I also did some additional tweaks in the margin that was given to these components to make it more consistent and less prone to breaking over time. 

**Note:** Before the CTA buttons were centered vertically. However, I think when the title is longer than 1 line it looks very ugly, so I took the liberty to align it to the top instead. 

Fixes #5025 

### Before

| Before | After |
|---|---|
| image | image |

**Environment**: Staging(main branch)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [ ] Go to the app on your ios phone, save the app to your homescreen, and go to an event-type detail page. Notice that the CTA buttons should be aligned properly now
- [ ] Check other pages where to CTA button is used to ensure nothing broke there (eg availability pages, workflow).

### CODE REVIEW

File: packages/ui/v2/core/Shell.tsx [hunk 1]
```diff
@@ -802,7 +802,7 @@ export function ShellMain(props: LayoutProps) {
               <div
                 className={classNames(
                   props.backPath ? "relative" : "fixed right-4 bottom-[75px] z-40 ",
-                  "cta mb-4 flex-shrink-0 sm:relative sm:bottom-auto sm:right-auto"
```
Comment (JeroenReumkens) [inline packages/ui/v2/core/Shell.tsx:805]: This global `cta` was actually the main culprit of messing things up. I also tested other pages where this Shell CTA was used, and couldn't find any occurrence where this seemed necessary.

**Comment by Udit-takkar** [comment]
i think i have also fixed this thing here https://github.com/calcom/cal.com/pull/5071

**Comment by JeroenReumkens** [comment]
Oh thanks for letting me know @Udit-takkar  When checking your PR I don't see the `cta` class removed, but perhaps you solved it differently indeed. Did you check in PWA mode specifically? 
I will block my changes until yours are merged, so I can test afterwards!

**Comment by Udit-takkar** [comment]
can you tell how to run PWA mode ?

**Comment by JeroenReumkens** [comment]
You can follow the steps outlined here to add the app to the homescreen (which is what we meant with PWA mode): https://www.macrumors.com/how-to/add-a-web-link-to-home-screen-iphone-ipad/ 

If you test this locally, you can access your local machine via your local network ip. After logging in I do get a redirect error when using the IP though, so locally I have to quite the PWA app after logging in, and then open again and I'm logged in. Just so you know 😉

**Comment by zomars** [comment]
I'm taking a look at the before and after and they seems both on non-paw mode. Also the global cat I think @joeauyeung has more context on this.

**Comment by Udit-takkar** [comment]
@JeroenReumkens i can't test PWA mode on android but i  tested with  local network ip on mobile browser and the buttons were aligned and design is a bit different now https://github.com/calcom/cal.com/pull/5071#issuecomment-1285407291

**Review by pumfleet — APPROVED** [review]

**Comment by JeroenReumkens** [comment]
Perfect @Udit-takkar Just get your PR merged, and I will merge main in mine and retest myself as well to see if there's any changes left from this PR that should still go in 😄

**Pull request merged**